modules/odc-creating-apiserversource.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * applications/application_life_cycle_management/odc-knative-event-sources.adoc
+
+[id="odc-creating-apiserversource_{context}"]
+= Creating an ApiServerSource event source using the Developer perspective
+
+.Procedure
+
+. Navigate to the **Add** page.
++
+image::event-sources-odc.png[Add an Event Source in the Developer perspective]
+. Select **Event Sources** -> **ApiServerSource**.
++
+image::apiserversource-odc.png[Create an ApiServerSource]
+. Configure the **ApiServerSource** settings:
++
+image::apiserver-odc-config-1.png[Configuring the ApiServerSource]
++
+.. Enter `sources.knative.dev/v1alpha1` as the **APIVERSION**, and `ApiServerSource` as the **KIND**.
+// TODO: make sure version is right across all serverless docs, if this needs to be updated, update in `kn` docs too.
+// .. What does 'mode' do?
+.. Select the **Service Account Name** for the service account that you created for event sources.
+.. Select the targeted Knative service from the dropdown menu in **Sink** -> **Knative Service**.
+// .. Add details about the image:
+// General -> Application Name, Name
+// image::apiserver-odc-config-2.png[Configuring the ApiServerSource]
+. Click **Create**.
+
+.Verification steps
+
+. After you have created the ApiServerSource, you will see it connected to the service it is sinked to in the **Topology** view.
++
+image::toplogy-odc-apiserver.png[ApiServerSource Topology view]
+
+// . Create events by launching a pod in the default namespace.
+// TODO: How to do this? What should a user see?
```

Comment:
to verify can follow steps as in here for Event Creation if watching events or can deploy image through `container image` in Add flow from ODC with image `busybox`

```
kubectl run busybox --image=busybox --restart=Never -- ls
kubectl delete pod busybox
``` 
https://github.com/knative/docs/tree/master/docs/eventing/samples/kubernetes-event-source#create-events

and check pods logs in devConsole can see events

<img width="1498" alt="Screenshot 2020-06-16 at 12 33 23 PM" src="https://user-images.githubusercontent.com/5129024/84742222-cd8dcc00-afcd-11ea-889d-a640fdc77159.png">

Reply:
Thank you!

Reply:
On second thoughts, I think this might be sort of out of scope for this PR. This is because verification steps are optional, and adding details about creating an ODC flow using busybox isn't really something we require. Thanks for the info anyway though, we could maybe add this in a future PR when there's more time to doc optional steps.